docs/paper/dev/event-api/event-listeners.md

```diff
@@ -0,0 +1,125 @@
+---
+slug: /dev/event-api/event-listeners
+---
+
+# Event Listeners
+
+Events are an efficient way to listen for specific actions that happen in the game. They can be called by the server, or by plugins. 
+These are called by the server when something happens, such as a player joining the server, or a block being broken. 
```

Comment:
This is a bit clunky with "These are called by the server" and "They can be called by plugins" breaking the flow/repeating what was said right before